learn/how-tos/dynamic-tabs-accordions.md

```diff
@@ -0,0 +1,132 @@
+---
+title: "Implementing Dynamic Tabs and Accordions"
+id: ""
+---
+---
+
+## Dynamic Tabs
+
+A Tab component is an essential part of every web application. WaveMaker provides a really flexible and powerful Tab component. It is possible to create a fully dynamic and flexible tab component without much effort using a data source.
```

Comment:
@SaraswathiRekhala can we also link to how Tabs can be created from static dataset as well? If there is no documentation for that, just add a line here with screenshot before talking about dynamic tabs

Reply:
@prasanth-reddy-wavemaker-com,  the usecase added shows working with model variable.